### ISSUE TITLE
Bug report - required xfce and Lxde desktop packages fails to install due to non-existant repositories

### ISSUE THREAD

**Issue by Throawayelt** [CypherpunkArmory/UserLAnd#1603]
# Describe the bug

Attempting to install Xfce or Lxde fails due to required packages not being found.

# Steps to reproduce the behavior:

1: Install either Xfce or Lxde ui terminal should begin package installation

2: Terminal will begin to spit out Errors due to packages no longer being found.

3: Installation should end prematurely.

# Screenshots

N/A

# Device Information

Device: Samsung Galaxy S10e.  
Android Version: 12.  
UserLAnd Version 3.0.0.  
Device is not rooted.  
Running stock OS

Please add any other context you think might help us resolve your issue.

(Problem was already posted but was flagged by bot for lacking info)
Linking additional repositories upon install to point towards needed packages might fix this.

**Comment by oskar-skog** [comment]
Is it on Debian?

https://github.com/CypherpunkArmory/UserLAnd/issues/1598

**Comment by Throawayelt** [comment]
Yes, trying to get around by editing the source list, but it's a bit of a hassle having to figure out all the required packages to try installing either or desktops.

**Comment by Throawayelt** [comment]
Yes, it is

**Comment by Throawayelt** [comment]
Ended up finding a workaround by just changing the source list using nano in a ssh session to use debian 11's repositories and upgrading the packages afterwards. Then starting an lxde session.

**Comment by Zoro0783** [comment]
I'm experiencing the same thing, I'm using debian, I'm failing to install R too, getting the same error

**Comment by drtrigon** [comment]
Same here wanting to install xfce desktop and firefox - both run into this issue.

Please update the repo info - thanks in advance!

**Comment by drtrigon** [comment]
> Ended up finding a workaround by just changing the source list using nano in a ssh session to use debian 11's repositories and upgrading the packages afterwards. Then starting an lxde session.

@Throawayelt: Could you give a bit more details on how you did this? (Preferably in a short step-by-step tutorial.)

**Comment by drtrigon** [comment]
> > Ended up finding a workaround by just changing the source list using nano in a ssh session to use debian 11's repositories and upgrading the packages afterwards. Then starting an lxde session.
> 
> @Throawayelt: Could you give a bit more details on how you did this? (Preferably in a short step-by-step tutorial.)

I can answer it myself now:
1. start "Debian"
2. run `$ sudo apt update`
3. exit and stop app
4. now the others like xfce desktop, firefox, etc. run w/o issues, e.g. the installation works and the apps start

**Comment by BenjaminWegener** [comment]
> > > Ended up finding a workaround by just changing the source list using nano in a ssh session to use debian 11's repositories and upgrading the packages afterwards. Then starting an lxde session.
> > 
> > 
> > @Throawayelt: Could you give a bit more details on how you did this? (Preferably in a short step-by-step tutorial.)
> 
> I can answer it myself now:
> 
> 1. start "Debian"
> 2. run `$ sudo apt update`
> 3. exit and stop app
> 4. now the others like xfce desktop, firefox, etc. run w/o issues, e.g. the installation works and the apps start

thanks, helped a lot. i had to `sudo apt update && sudo apt install whiptail -y && sudo apt install dbus dbus-x11 -y && sudo apt upgrade` to get rid of "no frontend dialog" and "cant connect to dbus" errors

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
